Data/Sys/GameSettings/SD2J01.ini

```diff
@@ -26,8 +26,8 @@ PH_ZNear =
 PH_ZFar =
 
 [Video_Settings]
-SafeTextureCacheColorSamples = 512
```

Comment:
Not needed anymore, this was true in the old revisions (3.5 and past) but builds from 4.0 era and current development versions output smooth intro videos and dancers regardless of safe texture cache setting...